modules/c/nrt/source/Utils.c

```diff
@@ -489,3 +489,30 @@ NRTPROT(void) nrt_Utils_decimalLonToGeoCharArray(double decimal, char *buffer8)
     nrt_Utils_decimalToGeographic(decimal, &d, &m, &s);
     nrt_Utils_geographicLonToCharArray(d, m, s, buffer8);
 }
+
+NRTAPI(void) nrt_Utils_byteSwap(nrt_Uint8 *value, nrt_Uint8 size)
+{
+    nrt_Uint8 temp;
+
+    switch(size)
+    {
+        case 2:
```

Comment:
Style-wise, these should all be brought in one indent level (`case` should start at the same column as the curly brace in the line above it)